oras/oci.py

```diff
@@ -128,7 +129,10 @@ def ManifestConfig(
     """
     # Create an empty config if we don't have one
     if not path or not os.path.exists(path):
-        path = "/dev/null"
+        if platform.system() == "Windows":
+            path = "nul"
+        else:
+            path = "/dev/null"
```

Comment:
Here you can do:

```python
path = "nul" if platform.system() == "Windows" else "/dev/null" 
```

Reply:
I found a better way to fix with os.devnull